### DISCUSSION TITLE
Is there a way to receive notifications when a pipeline fails?

### DISCUSSION THREAD

**Discussion by lucasg1** [elyra-ai/elyra#3157] [Q&A]
Hi everyone!

I've been using Elyra and KF for a few months now and I have one pipeline (created with Elyra) running daily for a while. One day I found that this pipeline failed some time ago and I only got to know a long time later, making it impossible to retrace the steps that the pipeline was supposed to do that day (the jobs are date dependent).

With that in mind, is it possible to receive email notifications when a pipeline fails?

I appreciate the work and your help, Elyra is awesome!

**Comment by lresende** [comment]
We delegate the execution of the pipelines to the runtimes, so this would be something on the airflow or kubeflow side.
